packages/agreement-process/src/services/agreementService.ts

```diff
@@ -1274,6 +1312,50 @@ export function agreementServiceBuilder(
 
       return updatedAgreement;
     },
+    async internalArchiveAgreement(
```

Comment:
Same comment as above
```suggestion
    async internalArchiveAgreementAfterDelegationRevocation(
```